client/src/reader/components/section/body-nodes/nodes/Math/annotationHelpers.js

```diff
@@ -103,9 +103,10 @@ export const getAnnotationStyles = (
   const interactiveAttributes =
```

Comment:
Most of this file is just duplicated from the logic in `Text` node. It would be lovely to extract it to a single place. Should we do that now?

Reply:
Sure seems like it!